integration_tests/cases/env/cluster/ddl/partition_table.sql

```diff
@@ -57,15 +57,15 @@ SELECT * from partition_table_t where name in ("horaedb5", "horaedb6", "horaedb7
 -- SQLNESS REPLACE duration=\d+.?\d*(µ|m|n) duration=xx
 -- SQLNESS REPLACE compute=\d+.?\d*(µ|m|n) compute=xx
 -- SQLNESS REPLACE time=\d+.?\d*(µ|m|n) time=xx
--- SQLNESS REPLACE metrics=\[.*?s\] metrics=xx
+-- SQLNESS REPLACE elapsed_compute=\d+.?\d*(µ|m|n) elapsed_compute=xx
 -- SQLNESS REPLACE scan_memtable_\d+ scan_memtable_n
 EXPLAIN ANALYZE SELECT * from partition_table_t where name = "ceresdb0";
 
 -- SQLNESS REPLACE duration=\d+.?\d*(µ|m|n) duration=xx
 -- SQLNESS REPLACE compute=\d+.?\d*(µ|m|n) compute=xx
 -- SQLNESS REPLACE __partition_table_t_\d __partition_table_t_x
 -- SQLNESS REPLACE time=\d+.?\d*(µ|m|n) time=xx
--- SQLNESS REPLACE metrics=\[.*?s\] metrics=xx
```

Comment:
This regular expression was used to replace duration value in metrics (metrics=[output_rows=0, elapsed_compute=15.5µs]). instead, it replaces more text which causing overwriting predicate metrics.

Orignal text:
plan_type,plan,\nString(\"Plan with Metrics\"),String(\"ResolvedPartitionedScan: pushdown_continue:false, partition_count:3, metrics=[\\npartition_table_t:\\n    __partition_table_t_0:\\n        poll_duration=65.875µs\\n        total_duration=1.732042ms\\n        wait_duration=1.666167ms\\n    __partition_table_t_1:\\n        poll_duration=44.5µs\\n        total_duration=2.275958ms\\n        wait_duration=2.231458ms\\n    __partition_table_t_3:\\n        poll_duration=69.249µs\\n        total_duration=2.367166ms\\n        wait_duration=2.297917ms\\n\\n__partition_table_t_0:\\nCoalescePartitionsExec, metrics=[output_rows=0, elapsed_compute=21.5µs]\\n  ScanTable: table=__partition_table_t_0, parallelism=8, priority=Low, partition_count=UnknownPartitioning(8), metrics=[\\nPredicate { exprs:[name IN ([Utf8(\\\"ceresdb2\\\")])], time_range:TimeRange { inclusive_start: Timestamp(-9223372036854775808), exclusive_end: Timestamp(9223372036854775807) } }\\nscan_table:\\n    do_merge_sort=true\\n    iter_num=1\\n    merge_iter_0:\\n        init_duration=162.042µs\\n        num_memtables=0\\n        num_ssts=0\\n        scan_count=1\\n        scan_duration=0ns\\n        times_fetch_row_from_multiple=0\\n        times_fetch_rows_from_one=0\\n        total_rows_fetch_from_one=0\\n        scan_memtable_28, fetched_columns:[tsid,t,name,id,value]:\\n=0]\\n\\n__partition_table_t_1:\\nCoalescePartitionsExec, metrics=[output_rows=0, elapsed_compute=24.791µs]\\n  ScanTable: table=__partition_table_t_1, parallelism=8, priority=Low, partition_count=UnknownPartitioning(8), metrics=[\\nPredicate { exprs:[name IN ([Utf8(\\\"ceresdb0\\\"), Utf8(\\\"ceresdb4\\\")])], time_range:TimeRange { inclusive_start: Timestamp(-9223372036854775808), exclusive_end: Timestamp(9223372036854775807) } }\\nscan_table:\\n    do_merge_sort=true\\n    iter_num=1\\n    merge_iter_0:\\n        init_duration=124.875µs\\n        num_memtables=0\\n        num_ssts=0\\n        scan_count=1\\n        scan_duration=42ns\\n        times_fetch_row_from_multiple=0\\n        times_fetch_rows_from_one=0\\n        total_rows_fetch_from_one=0\\n        scan_memtable_5, fetched_columns:[tsid,t,name,id,value]:\\n=0]\\n\\n__partition_table_t_3:\\nCoalescePartitionsExec, metrics=[output_rows=0, elapsed_compute=20.709µs]\\n  ScanTable: table=__partition_table_t_3, parallelism=8, priority=Low, partition_count=UnknownPartitioning(8), metrics=[\\nPredicate { exprs:[name IN ([Utf8(\\\"ceresdb1\\\"), Utf8(\\\"ceresdb3\\\")])], time_range:TimeRange { inclusive_start: Timestamp(-9223372036854775808), exclusive_end: Timestamp(9223372036854775807) } }\\nscan_table:\\n    do_merge_sort=true\\n    iter_num=1\\n    merge_iter_0:\\n        init_duration=162.458µs\\n        num_memtables=0\\n        num_ssts=0\\n        scan_count=1\\n        scan_duration=42ns\\n        times_fetch_row_from_multiple=0\\n        times_fetch_rows_from_one=0\\n        total_rows_fetch_from_one=0\\n        scan_memtable_1, fetched_columns:[tsid,t,name,id,value]:\\n=0]\\n=0]\\n\"),\n

Reply:
Good catch!